docker/docker.go

```diff
@@ -408,6 +431,20 @@ func newDockerImage(ctx context.Context, imageName string, option fanal_types.Do
 	return nil, func() {}, result
 }
 
+func newLocalDockerImage(imageName string) (containerregistry_v1.Image, error) {
+	ref, err := name.ParseReference(imageName, []name.Option{}...)
```

Comment:
what about
```
	if option.NonSSL {
		nameOpts = append(nameOpts, name.Insecure)
	}
```
options.NonSSL = config.InsecureRegistry

you are creating 
```
		opts := fanal_types.DockerOption{
			Timeout:               90 * time.Second,
			SkipPing:              true,
			UserName:              config.User,
			Password:              config.Password,
			NonSSL:                config.InsecureRegistry,
			InsecureSkipTLSVerify: config.InsecureTLS,
		}
```
for the `newRemoteDockerImage` - I think you can use it also here in `newLocalDockerImage`

Reply:
Not sure I understand, these options are relevant only for the registry. Here it's local. Are you talking about []name.Option{}...?

Reply:
yes I know, from `/Users/idanf/gocode/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20201028122920-48a25d865182/image/image.go` it seems that `name.Insecure` is related also to local (or maybe I'm wrong)
```
func newDockerImage(ctx context.Context, imageName string, option types.DockerOption) (v1.Image, func(), error) {
	var result error

	var nameOpts []name.Option
	if option.NonSSL {
		nameOpts = append(nameOpts, name.Insecure)
	}
	ref, err := name.ParseReference(imageName, nameOpts...)
	if err != nil {
		return nil, func() {}, xerrors.Errorf("failed to parse the image name: %w", err)
	}

	// Try accessing Docker Daemon
	img, cleanup, err := daemon.Image(ref)
```

Reply:
Not sure what it does, but no problem